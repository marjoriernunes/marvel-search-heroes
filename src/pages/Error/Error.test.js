import {render} from '@testing-library/react'
import ErrorScreen from './Error'

test('renders correctly', () =>{
    const utils = render(<ErrorScreen/>)
    expect(utils.container).toMatchSnapshot()
});

test('renders a error message', () => {
    const { getByText } = render(<ErrorScreen/>)
    expect(getByText('Oooops! A S.H.I.E.L.D não localizou nenhum herói com este nome em nosso banco de dados :(')).toBeInTheDocument()
});

