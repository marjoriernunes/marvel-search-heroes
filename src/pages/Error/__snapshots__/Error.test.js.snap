// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <p
    class="text-warning"
  >
    Oooops! A S.H.I.E.L.D não localizou nenhum herói com este nome em nosso banco de dados :(
     
  </p>
</div>
`;
